src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,87 @@
+package duke;
+
+
+import java.io.*;
```

Comment:
I believe you should state the classes explicitly instead of using *, perhaps your IDE was not set up properly?